src/polygons.jl

```diff
@@ -1564,7 +1564,7 @@ function polyhull(points)
     end
     # find a point with the highest Y coordinate value
 
-    if VERSION > v"1.5.0"
+    if VERSION ≥ v"1.7.0"
         _, anchorindex = findmax(pt -> pt.y, points)
```

Comment:
This `findmax` method is available from Julia v1.7.
https://docs.julialang.org/en/v1/base/collections/#Base.findmax